pinot-clients/pinot-java-client/src/test/java/org/apache/pinot/client/ControllerBrokerSelectorTest.java

```diff
@@ -0,0 +1,36 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.pinot.client;
+
+public class ControllerBrokerSelectorTest {
+  private ControllerBrokerSelectorTest() {
+  }
+
+  public static void main(String[] args) {
```

Comment:
We should be putting this inside `@Test` method and not main method.

Reply:
I had added this for quickly being able to test the controller based client. Similar to https://github.com/apache/pinot/blob/master/pinot-clients/pinot-java-client/src/test/java/org/apache/pinot/client/ConnectionFactoryTest.java#L110

Reply:
I guess we will need to remove from `ConnectionFactoryTest` class as well in a different PR. Doesn't make sense to have a main method in test.

Reply:
Removing this class. Will add integration tests

Reply:
Removing this class. Will add integration tests